doc/source/_static/js/landing_page.js

```diff
@@ -70,11 +74,17 @@ function displayFamilies(familyCounts) {
     familyName.className = "family-name";
     familyName.textContent = family;
 
+    const iconImage = document.createElement("img");
+    iconImage.alt = `${family} icon`;
+    iconImage.className = "ansys-icon";
+    iconImage.src = `_static/icons/${icon}`;
```

Comment:
Any chance we can set this as a global? Something like:

```suggestion
    iconImage.src = `${ICONS}/${icon}`;
```

This variable must be declared next to the `PROJECTS_FILE`.

Reply:
The variable is already declared in yaml file, added the path also in [d15d182](https://github.com/ansys/pyansys/pull/859/commits/d15d182b0650dfeb372ac99da6dc62d1fbe839e5)